Automattic/wp-calypso
# Plans: Replace "priority support" feature items with one large clause

#### Changes proposed in this Pull Request

Related to `1196108640073826-as-1199527862646257`.

* Remove "priority support" from all product and plan cards.
* Remove the `PRIORITY_SUPPORT_V2` feature object and slug, since they're no longer used.
* Add an asterisk clause to the middle of the product selector page, mentioning that all plans include priority support.

#### Testing instructions

* In any environment, open the Plans/Pricing page.
* Verify that product and plan cards no longer mention "priority support" in their list of features.
* Verify that a large asterisk clause is visible in the middle of the page stating "All plans include priority support" (see reference screenshot). Styling should match the mock-up at `zp0udhbEB9dvdS4rB3GJ0z-fi-923%3A0`.

#### Reference screenshot

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/670067/107793379-d8932a80-6d1b-11eb-99a3-f8e0ac8f14c8.png">
